deployer/cloudflaredeployer.go

```diff
@@ -0,0 +1,312 @@
+package deployer
+
+import (
+	"bytes"
+	"context"
+	_ "embed"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/pkg/errors"
+
+	"github.com/suborbital/atmo/atmo/appsource"
+	"github.com/suborbital/atmo/atmo/options"
+	"github.com/suborbital/atmo/directive"
+	subotemplate "github.com/suborbital/subo/builder/template"
+	"github.com/suborbital/subo/project"
+	"github.com/suborbital/subo/subo/util"
+)
+
+const (
+	cloudflareDeployJobType = "cloudflare"
+	routerScriptName        = "subo-router"
+
+	bundlePath = "./runnables.wasm.zip"
+)
+
+//go:embed cloudflare/shared.js
+var shared string
+
+//go:embed cloudflare/itty-router.js
```

Comment:
Is there any reason why these should be embedded instead of included in the `templates/cf` directory?